README.md

```diff
@@ -87,9 +87,17 @@ you must do.
         brew tap ArmMbed/homebrew-formulae
         brew install arm-none-eabi-gcc
 
+    On Windows Subsystem for Linux (WSL):
+
+        sudo add-apt-repository ppa:team-gcc-arm-embedded/ppa
+        sudo apt-get update
+        sudo apt-get install gcc-arm-embedded
+
+
 2. Install Nordic's [command line
    tools](https://infocenter.nordicsemi.com/index.jsp?topic=%2Fug_nrf5x_cltools%2FUG%2Fcltools%2Fnrf5x_installation.html)
    `mergehex` and `nrfjprog`. Ensure these tools are extracted and added to your path.
+   For WSL: install the `nrfjprog` for Windows, and not for Linux.
```

Comment:
Add a link to where they can find `nrfjprog`

Reply:
I'm afraid of links becoming dead-links, due to lack of maintenance. Do you feel strongly about this?

Reply:
Better to have a link that becomes dead in the future than to never have a link at all.

Reply:
Yeah, we will fix it as it becomes a problem

Reply:
Done.